<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryReadProperties</class>
 <widget class="QWidget" name="MemoryReadProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>135</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="chkAutomaticSize">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This option will automatically assign a size on each thread the task will use. The size will be less than the L3 cache of each thread and depending on how many threads will share this cache.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Automatic size for each thread (Topology aware)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblMemorySizePerThread">
     <property name="text">
      <string>Memory size per thread:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbMemorySize">
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> Bytes</string>
     </property>
     <property name="minimum">
      <number>256</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="singleStep">
      <number>256</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This batch will attempt to measure how much memory bandwidth is achievable from the system.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
